pages/index.tsx

```diff
@@ -107,9 +97,9 @@ const Home: React.FC<HomePageProps> = (props: HomePageProps) => {
 
 export default Home;
```

Comment:
If you're gonna do it, you gotta do it all

```suggestion
export default Index;
```